src/ui/components/Library/Team/View/TestRuns/Overview/TestResultListItem.tsx

```diff
@@ -50,17 +49,21 @@ export function TestResultListItem({
 
   label = label.toLowerCase();
 
-  let color;
+  let iconClass;
+  let iconType;
   switch (label) {
     case "failed":
-      color = "#EB5757";
+      iconClass = "failed";
+      iconType = "play-processed";
```

Comment:
Same comment from before about this type. It should be renamed to `test-suites-play-failed` and the other one to `test-suites-play-passing`